docs/decision-logs-guide/api/decisions.mdx

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_label: decisions
+---
+
+# decision_logs/decisions
+
+The `decisions` API opens a [WebSocket](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket) that receives
+decision events in near real time; events are typically no more than a minute old. 
+
+## WebSocket
+
+### Create a new decision logs WebSocket
+```javascript
+ws = new WebSocket("wss://.../api/v1/decision_logs/decisions/policy/<policy-id>",
```

Comment:
Instead of ... maybe use <aserto_authorizer_url> or something similar to indicate what the URL is expected to be?